components/merkledb/benches/storage.rs

```diff
@@ -76,7 +76,7 @@ fn plain_map_index_insert(b: &mut Bencher, len: usize) {
         |(db, data)| {
             let fork = db.fork();
             {
-                let mut table = MapIndex::new(NAME, &fork);
+                let mut table = fork.as_ref().ensure_map(NAME);
```

Comment:
`as_ref()` required because `Fork` doesn't implement `RawAccess`, `&Fork` does (which, in turn, is due to the fact that the associated type `ChangeSet` in `RawAccess` needs to be generic in order to allow `Fork` implementation). It is possible to write `(&fork).blah()` instead, but IMO `fork.as_ref().blah()` reads better.